reactioncommerce/reaction
# Fix console error when changing shops

Resolves #2727 

### Steps to test

1. Login as admin
1. Go to Shop settings/Marketplace and create a second shop
1. Using the shop switcher, switch to your second shop
1. Observe there are no errors and the page loads fine
